airflow/providers/amazon/aws/sensors/redshift.py

```diff
@@ -48,7 +48,7 @@ def __init__(
         self.hook: Optional[RedshiftHook] = None
 
     def poke(self, context):
-        self.log.info('Poking for status : %s\nfor cluster %s', self.target_status, self.cluster_identifier)
+        self.log.info('Checking cluster %r for status %r', self.cluster_identifier, self.target_status)
```

Comment:
probablly best to remove this change too since it's unrelated.  it made sense  when  you were changing everything to enum and touching this bit of code anyway.  but not anymore